### ISSUE TITLE
metric depth miss the .pth file

### ISSUE THREAD

**Issue by Donaure** [LiheYoung/Depth-Anything#34]
Thanks for your such a great job.
When I run `evaluate.py`, it raised an erro that  No such file or directory: '../checkpoints/depth_anything_vitl14.pth' in `metric_depth/zoedepth/models/base_models/depth_anything.py`
```python
depth_anything = DPT_DINOv2(out_channels=[256, 512, 1024, 1024], use_clstoken=False)
        
state_dict = torch.load('../checkpoints/depth_anything_vitl14.pth', map_location='cpu')

depth_anything.load_state_dict(state_dict)
```
Where can I find the file `depth_anything_vitl14.pth`?
On [https://huggingface.co/LiheYoung/depth_anything_vitl14/tree/main](https://huggingface.co/LiheYoung/depth_anything_vitl14/tree/main) I just find `depth_anything_vitl14.bin`.

**Comment by LiheYoung** [comment]
Oh. Thank you for your reminder! We have updated the README:https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth#training.

**Comment by lkyf** [comment]
Thanks, the problem is solved
